src/main/java/core/basesyntax/operations/ReturnOperation.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private final Storage storage;

```

Comment:
According to the encapsulation principle, the storage should not be directly exposed. Consider making it private and accessing it through methods if necessary.